.flake8

```diff
@@ -11,3 +11,5 @@ exclude =
     docker,
     # Ignore test data, which is annoying to properly format.
     tests/canned_responses/,
+per-file-ignores =
+    landoapi/secapproval.py:B950,W293
```

Comment:
It'd be a real shame to turn these off for the whole file, especially `W293`. See my other comment below, I'd prefer we work around it a different way.

Reply:
Done.